src/internal/types.ts

```diff
@@ -60,7 +67,7 @@ export interface TimeInterval<T> {
   interval: number;
 }
 
-/** SUBSCRIPTION INTERFACES */
+/* SUBSCRIPTION INTERFACES */
```

Comment:
Removing these prevents such comments to be included in the docs. [For example](https://rxjs.dev/api/index/interface/Unsubscribable):

<img width="453" alt="image" src="https://user-images.githubusercontent.com/28087049/157939186-e4f76cde-76b1-489a-aa49-c9dc568f056c.png">